crates/wash-lib/src/build/component.rs

```diff
@@ -434,7 +434,11 @@ async fn generate_tinygo_bindgen(project_dir: impl AsRef<Path>) -> Result<()> {
     let mut command = tokio::process::Command::new("go");
     let result = command
         .args(vec!["generate", "./..."])
-        .status()
+        // NOTE: this can be removed once upstream merges verbose flag
+        // https://github.com/bytecodealliance/wasm-tools-go/pull/214
+        .stderr(std::process::Stdio::piped())
+        .stderr(std::process::Stdio::piped())
```

Comment:
Should these be `stderr` and `stdout` instead of `stderr` twice?
```suggestion
        .stderr(std::process::Stdio::piped())
        .stdout(std::process::Stdio::piped())
```

Reply:
Thanks for this, was definitely a typo!